content/en/blog/falco-on-gke/index.md

```diff
@@ -0,0 +1,198 @@
+
+    ```
+    helm install falco \
+    -n falco \
+    --set tty=true \
+    --set driver.kind=ebpf \
+    falcosecurity/falco 
+    ```
+
+
+* Wait for the Falco pods to come online.
+
+    ```
+    kubectl get pods -n falco -w
+    ```
+
+    Eventually you should see something similar to this:
+
+    ```
+    falco-wfglg   2/2     Running           0          76s
+    falco-mdrlb   2/2     Running           0          91s
+    falco-7vxz6   2/2     Running           0          91s
+    ```
+
+    Note: You will see one Falco entry for each of the nodes in your cluster. In this case, Falco is running on a 3-node cluster, so there are 3 entries. 
+
+
+* Verify Falco is running correctly by examining the logs.
+
+
+    ```
+    kubectl logs -n falco -c falco -l app.kubernetes.io/name=falco`
+    ```
+    
+    You should see entries similar to this:
+
+    ```
+    Fri Nov  3 15:48:07 2023: Falco version: 0.36.2 (x86_64)
+    Fri Nov  3 15:48:07 2023: Falco initialized with configuration file: /etc/falco/falco.yaml
+    Fri Nov  3 15:48:07 2023: Loading rules from file /etc/falco/falco_rules.yaml
+    Fri Nov  3 15:48:07 2023: The chosen syscall buffer dimension is: 8388608 bytes (8 MBs)
+    Fri Nov  3 15:48:07 2023: Starting health webserver with threadiness 2, listening on port 8765
+    Fri Nov  3 15:48:07 2023: Loaded event sources: syscall
+    Fri Nov  3 15:48:07 2023: Enabled event sources: syscall
+    Fri Nov  3 15:48:07 2023: Opening 'syscall' source with BPF probe. BPF probe path: /root/.falco/falco-bpf.o
+
+    ```
+
+Falco is now successfully running on your GKE cluster. The next step is to simulate some suspicious activity and verify that Falco detects it. 
+
+## Testing Falco
+
+One of Falco’s default rules fires an alert if someone shells into a running container.  Follow the steps below to fire off that rule. 
+
+* Start an Alpine container and have it sleep so it stays running. 
+ 
+    ```
+    kubectl run alpine –image alpine – sh -c "sleep infinity"
+    ```
+
+* Execute a shell on the Alpine running container.  
+
+    ```
+    kubectl exec -it alpine -- sh -c "ls -al"
+    ``` 
+
+* Now check the Falco logs to see the alert. 
+
+    ```
+    kubectl logs -c falco -n falco -l app.kubernetes.io/name=falco |\
+    grep "Notice"
+    ```
+
+    You should see something like this:
+
+    ```
+    18:52:06.630209324: Notice A shell was spawned in a container with an attached terminal (evt_type=execve user=root user_uid=0 user_loginuid=-1 process=sh proc_exepath=/bin/busybox parent=runc command=sh -c ls -al terminal=34816 exe_flags=EXE_WRITABLE container_id=e71eac85a570 container_image=docker.io/library/alpine container_image_tag=latest container_name=alpine k8s_ns=default k8s_pod_name=alpine)
+    ``` 
+ 
+Notice all the data the alert provides including the container ID, image, and name, as well as the executed command. 
```

Comment:
I think `details` is a better term in this context than `data`. I would also indent according to the previous one.
```suggestion
    Notice all the details the alert provides including the container ID, image, and name, as well as the executed command. 
```